frontend/components/Course/Announcements.tsx

```diff
@@ -233,7 +234,23 @@ const AnnouncementMessage = ({
                             marginBottom: "0px",
                         }}
                     >
-                        {announcement.content}
+                        <Linkify
+                            componentDecorator={(
+                                decoratedHref: string,
+                                decoratedText: string,
+                                key: number
+                            ) => (
+                                <a
+                                    href={decoratedHref}
+                                    key={key}
+                                    target="_blank"
```

Comment:
If you want to add target="_blank" to open in new tab you may have to add rel="noopener noreferrer" for security reasons
https://perishablepress.com/fix-blank-target-vulnerability/ although modern browsers should automatically set this
https://mathiasbynens.github.io/rel-noopener/

Don't think it's a big deal if you don't put it (although it won't hurt), perhaps you can use this to catch TAs who are using internet explorer...